MAINTAINERS.md

```diff
@@ -0,0 +1,79 @@
+# Maintainer Guidelines
+
+**This guide is for maintainers.** These people have **write
+access** to NREL HPC's repository and help merge the contributions of
+others.
+
+If you have something to contribute, please see the [Contributing Guide](CONTRIBUTING.md).
+
+This is a living document - if you see something out of date or missing,
+speak up!
+
+## What are a maintainer's responsibilities?
+
+It is every maintainer's responsibility to:
+
+* Expose a clear roadmap for improving the repository.
+* Deliver prompt feedback and decisions on pull requests.
+* Be available to anyone with questions, bug reports, criticism etc. on the repository.
+  This includes GitHub issues and pull requests.
+* Make sure the repository respects the philosophy, design and roadmap of the project.
+
+## How are decisions made?
+
+This project is an open-source project with an open design philosophy. This
+means that the repository is the source of truth for EVERY aspect of the
+project, including its philosophy, design, and roadmap. *If it's
+part of the project, it's in the repo. It's in the repo, it's part of
+the project.*
+
+As a result, all decisions can be expressed as changes to the
+repository. 
+
+All decisions affecting this project, big and small, follow the same procedure:
+
+1. Open a pull request.
+   Anyone can do this.
+2. Discuss the pull request.
+   Anyone can do this.
+3. Review the pull request.
+   The relevant maintainers do this (see below [Who decides what?](#who-decides-what)).
+   Changes that affect project management (changing policy, cutting releases, etc.) are [proposed and voted on](GOVERNANCE.md).
+4. Merge or close the pull request.
+   The relevant maintainers do this.
+
+### I'm a maintainer, should I make pull requests too?
+
+Yes. Nobody should ever push to master directly. All changes should be
```

Comment:
We should add branch rules to prevent pushing to master and the gh-pages branch.